components/collective-page/hero/Hero.js

```diff
@@ -242,25 +244,25 @@ const Hero = ({ collective, host, isAdmin, onPrimaryColorChange }) => {
                       values={{
                         FiscalHost: <DefinedTerm term={Terms.FISCAL_HOST} color="black.700" />,
                         hostName: (
-                          <LinkCollective collective={host}>
-                            <Span data-cy="fiscalHostName" color="black.700">
-                              {host.name}
+                          <LinkCollective collective={host} data-cy="fiscalHostName">
+                            <Span color="black.700">
+                              <TruncatedTextWithTooltip value={host.name} />
                             </Span>
                           </LinkCollective>
                         ),
                       }}
                     />
                   </Container>
-                  {collective.connectedTo.length !== 0 && (
+                  {!parentIsHost && collective.connectedTo?.length > 0 && (
                     <Container mx={1} color="black.700" my="12px">
                       <FormattedMessage
                         id="Collective.Hero.ParentCollective"
                         defaultMessage="Part of: {parentName}"
                         values={{
                           parentName: (
-                            <LinkCollective collective={collective.connectedTo[0].collective}>
-                              <Span data-cy="parentCollectiveName" color="black.700">
-                                {collective.connectedTo[0].collective.name}
+                            <LinkCollective collective={collective}>
+                              <Span color="black.700" data-cy="parentCollectiveName">
+                                <TruncatedTextWithTooltip collective={collective.connectedTo[0].collective.name} />
```

Comment:
This is probably not working, as `TruncatedTextWithTooltip` expects a `value` prop, not `collective`

Reply:
Thanks I missed this because I'm not sure of a good host that isn't a parent. Do you know of one off the top of your head?

Reply:
@simonv3: yes `propTypes` does catch this and the error should be printed on the console. 🤔